joomla/joomla-cms
# PHPDoc for Json input

### Summary of Changes

Make IDE happy with the code like `$app->input->json->getRaw()`

### Testing Instructions

Apply patch, see IDE tips. The json input is mostly used in API controllers.

A PR is also required for Input package: https://github.com/joomla-framework/input/pull/40

### Actual result BEFORE applying this Pull Request

`$app->input->json` and `$app->input->json->getRaw()` are incorrectly parsed by IDE.

### Expected result AFTER applying this Pull Request

`$app->input->json` and `$app->input->json->getRaw()` are correctly parsed by IDE.
+
mistyped Json input usage in application config model.
